### DISCUSSION TITLE
Typo in enum shieldgren_e values?

### DISCUSSION THREAD

**Discussion by spodlesniy** [rehlds/ReGameDLL_CS#837] [General]
https://github.com/s1lentq/ReGameDLL_CS/blob/94bf7051968de517a2b72080c2c5ee15928cf6a8/regamedll/dlls/weapontype.h#L335-L340
Is is typo in enum values? Should it be SHIELD***G***REN_IDLE, etc?

**Comment by dystopm** [comment]
@spodlesniy check #843
